docs/src/model_components/carbon-chemistry.md

```diff
@@ -0,0 +1,350 @@
+# [Carbon Chemistry](@id carbon-chemistry)
+
+Our carbon chemistry model follows the best practice described by [dickson2007](@citet), as implemented by e.g. `cbsyst` [branson2023](@citep) and `CO2SYS` [humphreys2022](@citep) in Python.
+
+The carbon chemistry model is primarily used for diagnosing the partial pressure of carbon dioxide in the surface water for [gas exchange with the air](@ref air-sea-gas), but is capable of diagnosing other species such as carbonate concentration, useful in the calculation of calcite dissolution.
+The model works by computing the pH from the total dissolved inorganic carbon and total alkalinity (and optionally silicate, phosphate, boron, sulfate, and fluoride concentration), along with temperature and salinity, which can then be used to find the concentration of different species.
+We will first describe how to use the model, followed by the underlying chemistry and parameterisations.
+
+## Using the model
+To use the carbon chemistry model it is constructed by simply writing:
+```@example carbon-chem
+using OceanBioME
+
+carbon_chemistry = CarbonChemistry()
+```
+
+### Computing ``fCO_2`` and ``pH``
+To compute the fugacity of carbon dioxide (``fCO_2``) you call the model with the `DIC`, `Alk`alinity, `T`emperature, and `S`alinity:
```

Comment:
Do we want to label DIC, Alk, T and S in input instead of relying on the argument order?  DIC and Alk have similar values, so using the order could cause confusion. I think that this would also allow us to use multiple dispatch to handle the case where pH is specified and we wouldn't have to send in NaN for DIC or Alk.

Reply:
This is a good idea and I'll change it now